Failure test cases of DCL

TODO:

- [x] CREATE USER parse error (#6664)
- [x] Fix xml format error
- [x] Reduce time cost from 1 hour to  5 minutes
- [x] Clean ROLE resources for each test

## Bug Report

**For English only**, other languages will not accept.

Before report a bug, make sure you have:

- Searched open and closed [GitHub issues](https://github.com/apache/shardingsphere/issues).
- Read documentation: [ShardingSphere Doc](https://shardingsphere.apache.org/document/current/en/overview).

Please pay attention on issues you submitted, because we maybe need more details. 
If no response anymore and we cannot reproduce it on current information, we will **close it**.

Please answer these questions before submitting your issue. Thanks!

### Which version of ShardingSphere did you use?
built from master branch

### Which project did you use? ShardingSphere-JDBC or ShardingSphere-Proxy?
ShardingSphere-JDBC

### Expected behavior
Integration test using MySQL instead of H2. Expect all test cases pass.

### Actual behavior
There're several DCL cases failed.

### Reason analyze (If you can)

### Steps to reproduce the behavior, such as: SQL to execute, sharding rule configuration, when exception occur etc.

1. env.properties:
databases=MySQL

2. hosts: (suppose your MySQL is on localhost)
add 127.0.0.1 db.mysql

3. mvn clean install

### Example codes for reproduce this issue (such as a github link).

```
[INFO] Scanning for projects...
[INFO] ------------------------------------------------------------------------
[INFO] Detecting the operating system and CPU architecture
[INFO] ------------------------------------------------------------------------
[INFO] os.detected.name: windows
[INFO] os.detected.arch: x86_64
[INFO] os.detected.version: 10.0
[INFO] os.detected.version.major: 10
[INFO] os.detected.version.minor: 0
[INFO] os.detected.classifier: windows-x86_64
[INFO] 
[INFO] --------< org.apache.shardingsphere:shardingsphere-test-suite >---------
[INFO] Building shardingsphere-test-suite 5.0.0-RC1-SNAPSHOT
[INFO] --------------------------------[ jar ]---------------------------------
[INFO] 
[INFO] --- maven-enforcer-plugin:1.4:enforce (enforce-maven-version) @ shardingsphere-test-suite ---
[INFO] 
[INFO] --- maven-enforcer-plugin:1.4:enforce (enforce-banned-dependencies) @ shardingsphere-test-suite ---
[INFO] 
[INFO] --- maven-checkstyle-plugin:3.1.0:check (validate) @ shardingsphere-test-suite ---
[INFO] 
[INFO] --- maven-resources-plugin:2.7:copy-resources (copy-resources) @ shardingsphere-test-suite ---
[INFO] Using 'UTF-8' encoding to copy filtered resources.
[INFO] Copying 578 resources
[INFO] 
[INFO] --- jacoco-maven-plugin:0.8.5:prepare-agent (default) @ shardingsphere-test-suite ---
[INFO] 
[INFO] --- maven-remote-resources-plugin:1.5:process (process-resource-bundles) @ shardingsphere-test-suite ---
[INFO] 
[INFO] --- maven-resources-plugin:2.7:resources (default-resources) @ shardingsphere-test-suite ---
[INFO] Using 'UTF-8' encoding to copy filtered resources.
[INFO] Copying 3 resources
[INFO] 
[INFO] --- maven-compiler-plugin:3.3:compile (default-compile) @ shardingsphere-test-suite ---
[INFO] No sources to compile
[INFO] 
[INFO] --- maven-resources-plugin:2.7:testResources (default-testResources) @ shardingsphere-test-suite ---
[INFO] Using 'UTF-8' encoding to copy filtered resources.
[INFO] Copying 578 resources
[INFO] Copying 3 resources
[INFO] 
[INFO] --- maven-compiler-plugin:3.3:testCompile (default-testCompile) @ shardingsphere-test-suite ---
[INFO] Nothing to compile - all classes are up to date
[INFO] 
[INFO] --- maven-surefire-plugin:2.22.0:test (default-test) @ shardingsphere-test-suite ---
[INFO] 
[INFO] -------------------------------------------------------
[INFO]  T E S T S
[INFO] -------------------------------------------------------
[INFO] 
[INFO] Results:
[INFO] 
[INFO] Tests run: 0, Failures: 0, Errors: 0, Skipped: 0
[INFO] 
[INFO] 
[INFO] --- jacoco-maven-plugin:0.8.5:report (report) @ shardingsphere-test-suite ---
[INFO] Analyzed bundle 'shardingsphere-test-suite' with 0 classes
[INFO] 
[INFO] --- maven-jar-plugin:2.6:jar (default-jar) @ shardingsphere-test-suite ---
[INFO] 
[INFO] --- maven-javadoc-plugin:2.10.3:jar (attach-javadocs) @ shardingsphere-test-suite ---
[INFO] 
[INFO] --- maven-site-plugin:3.4:attach-descriptor (attach-descriptor) @ shardingsphere-test-suite ---
[INFO] 
[INFO] --- maven-failsafe-plugin:2.22.0:integration-test (integration-tests) @ shardingsphere-test-suite ---
[INFO] 
[INFO] -------------------------------------------------------
[INFO]  T E S T S
[INFO] -------------------------------------------------------
[INFO] Running org.apache.shardingsphere.dbtest.engine.dcl.GeneralDCLIT
[INFO] ======= Test Plan =======
[WARN] Total: 3760, Active: MySQL(416), Disabled: PostgreSQL(1664), Oracle(832), SQLServer(848) 

line 1:38 mismatched input '<EOF>' expecting {',', DEFAULT, DISCARD, IDENTIFIED}
line 1:38 mismatched input '<EOF>' expecting {',', DEFAULT, DISCARD, IDENTIFIED}
line 1:38 mismatched input '<EOF>' expecting {',', DEFAULT, DISCARD, IDENTIFIED}
line 1:38 mismatched input '<EOF>' expecting {',', DEFAULT, DISCARD, IDENTIFIED}
line 1:38 mismatched input '<EOF>' expecting {',', DEFAULT, DISCARD, IDENTIFIED}
line 1:38 mismatched input '<EOF>' expecting {',', DEFAULT, DISCARD, IDENTIFIED}
line 1:38 mismatched input '<EOF>' expecting {',', DEFAULT, DISCARD, IDENTIFIED}
line 1:38 mismatched input '<EOF>' expecting {',', DEFAULT, DISCARD, IDENTIFIED}
line 1:38 mismatched input '<EOF>' expecting {',', DEFAULT, DISCARD, IDENTIFIED}
line 1:65 mismatched input '<EOF>' expecting {',', DEFAULT}
line 1:65 mismatched input '<EOF>' expecting {',', DEFAULT}
line 1:65 mismatched input '<EOF>' expecting {',', DEFAULT}
line 1:65 mismatched input '<EOF>' expecting {',', DEFAULT}
line 1:65 mismatched input '<EOF>' expecting {',', DEFAULT}
line 1:65 mismatched input '<EOF>' expecting {',', DEFAULT}
line 1:65 mismatched input '<EOF>' expecting {',', DEFAULT}
line 1:65 mismatched input '<EOF>' expecting {',', DEFAULT}
line 1:65 mismatched input '<EOF>' expecting {',', DEFAULT}
[ERROR] Tests run: 416, Failures: 0, Errors: 63, Skipped: 0, Time elapsed: 4,355.159 s <<< FAILURE! - in org.apache.shardingsphere.dbtest.engine.dcl.GeneralDCLIT
[ERROR] assertExecute[db -> MySQL -> Literal -> CREATE ROLE role1](org.apache.shardingsphere.dbtest.engine.dcl.GeneralDCLIT)  Time elapsed: 3.003 s  <<< ERROR!
java.sql.SQLException: Operation CREATE ROLE failed for 'role1'@'%'
	at org.apache.shardingsphere.dbtest.engine.dcl.GeneralDCLIT.assertExecute(GeneralDCLIT.java:62)

[ERROR] assertExecuteUpdate[tbl -> MySQL -> Literal -> CREATE ROLE role1](org.apache.shardingsphere.dbtest.engine.dcl.GeneralDCLIT)  Time elapsed: 1.741 s  <<< ERROR!
java.sql.SQLException: Operation CREATE ROLE failed for 'role1'@'%'
	at org.apache.shardingsphere.dbtest.engine.dcl.GeneralDCLIT.assertExecuteUpdate(GeneralDCLIT.java:51)

[ERROR] assertExecute[tbl -> MySQL -> Literal -> CREATE ROLE role1](org.apache.shardingsphere.dbtest.engine.dcl.GeneralDCLIT)  Time elapsed: 1.98 s  <<< ERROR!
java.sql.SQLException: Operation CREATE ROLE failed for 'role1'@'%'
	at org.apache.shardingsphere.dbtest.engine.dcl.GeneralDCLIT.assertExecute(GeneralDCLIT.java:62)

[ERROR] assertExecuteUpdate[dbtbl_with_masterslave -> MySQL -> Literal -> CREATE ROLE role1](org.apache.shardingsphere.dbtest.engine.dcl.GeneralDCLIT)  Time elapsed: 31.275 s  <<< ERROR!
java.sql.SQLException: Operation CREATE ROLE failed for 'role1'@'%'
	at org.apache.shardingsphere.dbtest.engine.dcl.GeneralDCLIT.assertExecuteUpdate(GeneralDCLIT.java:51)

[ERROR] assertExecute[dbtbl_with_masterslave -> MySQL -> Literal -> CREATE ROLE role1](org.apache.shardingsphere.dbtest.engine.dcl.GeneralDCLIT)  Time elapsed: 30.508 s  <<< ERROR!
java.sql.SQLException: Operation CREATE ROLE failed for 'role1'@'%'
	at org.apache.shardingsphere.dbtest.engine.dcl.GeneralDCLIT.assertExecute(GeneralDCLIT.java:62)

[ERROR] assertExecuteUpdate[masterslave -> MySQL -> Literal -> CREATE ROLE role1](org.apache.shardingsphere.dbtest.engine.dcl.GeneralDCLIT)  Time elapsed: 3.176 s  <<< ERROR!
java.sql.SQLException: Operation CREATE ROLE failed for 'role1'@'%'
	at org.apache.shardingsphere.dbtest.engine.dcl.GeneralDCLIT.assertExecuteUpdate(GeneralDCLIT.java:51)

[ERROR] assertExecute[masterslave -> MySQL -> Literal -> CREATE ROLE role1](org.apache.shardingsphere.dbtest.engine.dcl.GeneralDCLIT)  Time elapsed: 3.855 s  <<< ERROR!
java.sql.SQLException: Operation CREATE ROLE failed for 'role1'@'%'
	at org.apache.shardingsphere.dbtest.engine.dcl.GeneralDCLIT.assertExecute(GeneralDCLIT.java:62)

[ERROR] assertExecuteUpdate[db -> MySQL -> Placeholder -> CREATE ROLE role1](org.apache.shardingsphere.dbtest.engine.dcl.GeneralDCLIT)  Time elapsed: 3.392 s  <<< ERROR!
java.sql.SQLException: Operation CREATE ROLE failed for 'role1'@'%'
	at org.apache.shardingsphere.dbtest.engine.dcl.GeneralDCLIT.assertExecuteUpdate(GeneralDCLIT.java:53)

[ERROR] assertExecute[db -> MySQL -> Placeholder -> CREATE ROLE role1](org.apache.shardingsphere.dbtest.engine.dcl.GeneralDCLIT)  Time elapsed: 3.167 s  <<< ERROR!
java.sql.SQLException: Operation CREATE ROLE failed for 'role1'@'%'
	at org.apache.shardingsphere.dbtest.engine.dcl.GeneralDCLIT.assertExecute(GeneralDCLIT.java:64)

[ERROR] assertExecuteUpdate[tbl -> MySQL -> Placeholder -> CREATE ROLE role1](org.apache.shardingsphere.dbtest.engine.dcl.GeneralDCLIT)  Time elapsed: 1.973 s  <<< ERROR!
java.sql.SQLException: Operation CREATE ROLE failed for 'role1'@'%'
	at org.apache.shardingsphere.dbtest.engine.dcl.GeneralDCLIT.assertExecuteUpdate(GeneralDCLIT.java:53)

[ERROR] assertExecute[tbl -> MySQL -> Placeholder -> CREATE ROLE role1](org.apache.shardingsphere.dbtest.engine.dcl.GeneralDCLIT)  Time elapsed: 2.062 s  <<< ERROR!
java.sql.SQLException: Operation CREATE ROLE failed for 'role1'@'%'
	at org.apache.shardingsphere.dbtest.engine.dcl.GeneralDCLIT.assertExecute(GeneralDCLIT.java:64)

[ERROR] assertExecuteUpdate[dbtbl_with_masterslave -> MySQL -> Placeholder -> CREATE ROLE role1](org.apache.shardingsphere.dbtest.engine.dcl.GeneralDCLIT)  Time elapsed: 30.744 s  <<< ERROR!
java.sql.SQLException: Operation CREATE ROLE failed for 'role1'@'%'
	at org.apache.shardingsphere.dbtest.engine.dcl.GeneralDCLIT.assertExecuteUpdate(GeneralDCLIT.java:53)

[ERROR] assertExecute[dbtbl_with_masterslave -> MySQL -> Placeholder -> CREATE ROLE role1](org.apache.shardingsphere.dbtest.engine.dcl.GeneralDCLIT)  Time elapsed: 31.559 s  <<< ERROR!
java.sql.SQLException: Operation CREATE ROLE failed for 'role1'@'%'
	at org.apache.shardingsphere.dbtest.engine.dcl.GeneralDCLIT.assertExecute(GeneralDCLIT.java:64)

[ERROR] assertExecuteUpdate[masterslave -> MySQL -> Placeholder -> CREATE ROLE role1](org.apache.shardingsphere.dbtest.engine.dcl.GeneralDCLIT)  Time elapsed: 4.404 s  <<< ERROR!
java.sql.SQLException: Operation CREATE ROLE failed for 'role1'@'%'
	at org.apache.shardingsphere.dbtest.engine.dcl.GeneralDCLIT.assertExecuteUpdate(GeneralDCLIT.java:53)

[ERROR] assertExecute[masterslave -> MySQL -> Placeholder -> CREATE ROLE role1](org.apache.shardingsphere.dbtest.engine.dcl.GeneralDCLIT)  Time elapsed: 3.334 s  <<< ERROR!
java.sql.SQLException: Operation CREATE ROLE failed for 'role1'@'%'
	at org.apache.shardingsphere.dbtest.engine.dcl.GeneralDCLIT.assertExecute(GeneralDCLIT.java:64)

[ERROR] assertExecuteUpdate[db -> MySQL -> Literal -> ALTER USER 'user1'@'localhost'](org.apache.shardingsphere.dbtest.engine.dcl.GeneralDCLIT)  Time elapsed: 3.154 s  <<< ERROR!
java.sql.SQLException: Operation ALTER USER failed for 'user1'@'localhost'
	at org.apache.shardingsphere.dbtest.engine.dcl.GeneralDCLIT.assertExecuteUpdate(GeneralDCLIT.java:51)

[ERROR] assertExecute[db -> MySQL -> Literal -> ALTER USER 'user1'@'localhost'](org.apache.shardingsphere.dbtest.engine.dcl.GeneralDCLIT)  Time elapsed: 3.46 s  <<< ERROR!
java.sql.SQLException: Operation ALTER USER failed for 'user1'@'localhost'
	at org.apache.shardingsphere.dbtest.engine.dcl.GeneralDCLIT.assertExecute(GeneralDCLIT.java:62)

[ERROR] assertExecuteUpdate[tbl -> MySQL -> Literal -> ALTER USER 'user1'@'localhost'](org.apache.shardingsphere.dbtest.engine.dcl.GeneralDCLIT)  Time elapsed: 2.012 s  <<< ERROR!
java.sql.SQLException: Operation ALTER USER failed for 'user1'@'localhost'
	at org.apache.shardingsphere.dbtest.engine.dcl.GeneralDCLIT.assertExecuteUpdate(GeneralDCLIT.java:51)

[ERROR] assertExecute[tbl -> MySQL -> Literal -> ALTER USER 'user1'@'localhost'](org.apache.shardingsphere.dbtest.engine.dcl.GeneralDCLIT)  Time elapsed: 1.702 s  <<< ERROR!
java.sql.SQLException: Operation ALTER USER failed for 'user1'@'localhost'
	at org.apache.shardingsphere.dbtest.engine.dcl.GeneralDCLIT.assertExecute(GeneralDCLIT.java:62)

[ERROR] assertExecuteUpdate[dbtbl_with_masterslave -> MySQL -> Literal -> ALTER USER 'user1'@'localhost'](org.apache.shardingsphere.dbtest.engine.dcl.GeneralDCLIT)  Time elapsed: 39.027 s  <<< ERROR!
java.sql.SQLException: Operation ALTER USER failed for 'user1'@'localhost'
	at org.apache.shardingsphere.dbtest.engine.dcl.GeneralDCLIT.assertExecuteUpdate(GeneralDCLIT.java:51)

[ERROR] assertExecute[dbtbl_with_masterslave -> MySQL -> Literal -> ALTER USER 'user1'@'localhost'](org.apache.shardingsphere.dbtest.engine.dcl.GeneralDCLIT)  Time elapsed: 31.861 s  <<< ERROR!
java.sql.SQLException: Operation ALTER USER failed for 'user1'@'localhost'
	at org.apache.shardingsphere.dbtest.engine.dcl.GeneralDCLIT.assertExecute(GeneralDCLIT.java:62)

[ERROR] assertExecuteUpdate[masterslave -> MySQL -> Literal -> ALTER USER 'user1'@'localhost'](org.apache.shardingsphere.dbtest.engine.dcl.GeneralDCLIT)  Time elapsed: 3.858 s  <<< ERROR!
java.sql.SQLException: Operation ALTER USER failed for 'user1'@'localhost'
	at org.apache.shardingsphere.dbtest.engine.dcl.GeneralDCLIT.assertExecuteUpdate(GeneralDCLIT.java:51)

[ERROR] assertExecute[masterslave -> MySQL -> Literal -> ALTER USER 'user1'@'localhost'](org.apache.shardingsphere.dbtest.engine.dcl.GeneralDCLIT)  Time elapsed: 3.251 s  <<< ERROR!
java.sql.SQLException: Operation ALTER USER failed for 'user1'@'localhost'
	at org.apache.shardingsphere.dbtest.engine.dcl.GeneralDCLIT.assertExecute(GeneralDCLIT.java:62)

[ERROR] assertExecuteUpdate[db -> MySQL -> Placeholder -> ALTER USER 'user1'@'localhost'](org.apache.shardingsphere.dbtest.engine.dcl.GeneralDCLIT)  Time elapsed: 4.532 s  <<< ERROR!
java.sql.SQLException: Operation ALTER USER failed for 'user1'@'localhost'
	at org.apache.shardingsphere.dbtest.engine.dcl.GeneralDCLIT.assertExecuteUpdate(GeneralDCLIT.java:53)

[ERROR] assertExecute[db -> MySQL -> Placeholder -> ALTER USER 'user1'@'localhost'](org.apache.shardingsphere.dbtest.engine.dcl.GeneralDCLIT)  Time elapsed: 3.439 s  <<< ERROR!
java.sql.SQLException: Operation ALTER USER failed for 'user1'@'localhost'
	at org.apache.shardingsphere.dbtest.engine.dcl.GeneralDCLIT.assertExecute(GeneralDCLIT.java:64)

[ERROR] assertExecuteUpdate[tbl -> MySQL -> Placeholder -> ALTER USER 'user1'@'localhost'](org.apache.shardingsphere.dbtest.engine.dcl.GeneralDCLIT)  Time elapsed: 1.705 s  <<< ERROR!
java.sql.SQLException: Operation ALTER USER failed for 'user1'@'localhost'
	at org.apache.shardingsphere.dbtest.engine.dcl.GeneralDCLIT.assertExecuteUpdate(GeneralDCLIT.java:53)

[ERROR] assertExecute[tbl -> MySQL -> Placeholder -> ALTER USER 'user1'@'localhost'](org.apache.shardingsphere.dbtest.engine.dcl.GeneralDCLIT)  Time elapsed: 2.148 s  <<< ERROR!
java.sql.SQLException: Operation ALTER USER failed for 'user1'@'localhost'
	at org.apache.shardingsphere.dbtest.engine.dcl.GeneralDCLIT.assertExecute(GeneralDCLIT.java:64)

[ERROR] assertExecuteUpdate[dbtbl_with_masterslave -> MySQL -> Placeholder -> ALTER USER 'user1'@'localhost'](org.apache.shardingsphere.dbtest.engine.dcl.GeneralDCLIT)  Time elapsed: 31.871 s  <<< ERROR!
java.sql.SQLException: Operation ALTER USER failed for 'user1'@'localhost'
	at org.apache.shardingsphere.dbtest.engine.dcl.GeneralDCLIT.assertExecuteUpdate(GeneralDCLIT.java:53)

[ERROR] assertExecute[dbtbl_with_masterslave -> MySQL -> Placeholder -> ALTER USER 'user1'@'localhost'](org.apache.shardingsphere.dbtest.engine.dcl.GeneralDCLIT)  Time elapsed: 30.816 s  <<< ERROR!
java.sql.SQLException: Operation ALTER USER failed for 'user1'@'localhost'
	at org.apache.shardingsphere.dbtest.engine.dcl.GeneralDCLIT.assertExecute(GeneralDCLIT.java:64)

[ERROR] assertExecuteUpdate[masterslave -> MySQL -> Placeholder -> ALTER USER 'user1'@'localhost'](org.apache.shardingsphere.dbtest.engine.dcl.GeneralDCLIT)  Time elapsed: 3.53 s  <<< ERROR!
java.sql.SQLException: Operation ALTER USER failed for 'user1'@'localhost'
	at org.apache.shardingsphere.dbtest.engine.dcl.GeneralDCLIT.assertExecuteUpdate(GeneralDCLIT.java:53)

[ERROR] assertExecute[masterslave -> MySQL -> Placeholder -> ALTER USER 'user1'@'localhost'](org.apache.shardingsphere.dbtest.engine.dcl.GeneralDCLIT)  Time elapsed: 3.787 s  <<< ERROR!
java.sql.SQLException: Operation ALTER USER failed for 'user1'@'localhost'
	at org.apache.shardingsphere.dbtest.engine.dcl.GeneralDCLIT.assertExecute(GeneralDCLIT.java:64)

[ERROR] assertExecuteUpdate[db -> MySQL -> Literal -> DROP ROLE role1](org.apache.shardingsphere.dbtest.engine.dcl.GeneralDCLIT)  Time elapsed: 3.167 s  <<< ERROR!
java.sql.SQLException: Operation DROP ROLE failed for 'role1'@'%'
	at org.apache.shardingsphere.dbtest.engine.dcl.GeneralDCLIT.assertExecuteUpdate(GeneralDCLIT.java:51)

[ERROR] assertExecute[db -> MySQL -> Literal -> DROP ROLE role1](org.apache.shardingsphere.dbtest.engine.dcl.GeneralDCLIT)  Time elapsed: 3.232 s  <<< ERROR!
java.sql.SQLException: Operation DROP ROLE failed for 'role1'@'%'
	at org.apache.shardingsphere.dbtest.engine.dcl.GeneralDCLIT.assertExecute(GeneralDCLIT.java:62)

[ERROR] assertExecuteUpdate[tbl -> MySQL -> Literal -> DROP ROLE role1](org.apache.shardingsphere.dbtest.engine.dcl.GeneralDCLIT)  Time elapsed: 2.164 s  <<< ERROR!
java.sql.SQLException: Operation DROP ROLE failed for 'role1'@'%'
	at org.apache.shardingsphere.dbtest.engine.dcl.GeneralDCLIT.assertExecuteUpdate(GeneralDCLIT.java:51)

[ERROR] assertExecute[tbl -> MySQL -> Literal -> DROP ROLE role1](org.apache.shardingsphere.dbtest.engine.dcl.GeneralDCLIT)  Time elapsed: 2.051 s  <<< ERROR!
java.sql.SQLException: Operation DROP ROLE failed for 'role1'@'%'
	at org.apache.shardingsphere.dbtest.engine.dcl.GeneralDCLIT.assertExecute(GeneralDCLIT.java:62)

[ERROR] assertExecuteUpdate[dbtbl_with_masterslave -> MySQL -> Literal -> DROP ROLE role1](org.apache.shardingsphere.dbtest.engine.dcl.GeneralDCLIT)  Time elapsed: 30.055 s  <<< ERROR!
java.sql.SQLException: Operation DROP ROLE failed for 'role1'@'%'
	at org.apache.shardingsphere.dbtest.engine.dcl.GeneralDCLIT.assertExecuteUpdate(GeneralDCLIT.java:51)

[ERROR] assertExecute[dbtbl_with_masterslave -> MySQL -> Literal -> DROP ROLE role1](org.apache.shardingsphere.dbtest.engine.dcl.GeneralDCLIT)  Time elapsed: 31.709 s  <<< ERROR!
java.sql.SQLException: Operation DROP ROLE failed for 'role1'@'%'
	at org.apache.shardingsphere.dbtest.engine.dcl.GeneralDCLIT.assertExecute(GeneralDCLIT.java:62)

[ERROR] assertExecuteUpdate[masterslave -> MySQL -> Literal -> DROP ROLE role1](org.apache.shardingsphere.dbtest.engine.dcl.GeneralDCLIT)  Time elapsed: 4.001 s  <<< ERROR!
java.sql.SQLException: Operation DROP ROLE failed for 'role1'@'%'
	at org.apache.shardingsphere.dbtest.engine.dcl.GeneralDCLIT.assertExecuteUpdate(GeneralDCLIT.java:51)

[ERROR] assertExecute[masterslave -> MySQL -> Literal -> DROP ROLE role1](org.apache.shardingsphere.dbtest.engine.dcl.GeneralDCLIT)  Time elapsed: 3.397 s  <<< ERROR!
java.sql.SQLException: Operation DROP ROLE failed for 'role1'@'%'
	at org.apache.shardingsphere.dbtest.engine.dcl.GeneralDCLIT.assertExecute(GeneralDCLIT.java:62)

[ERROR] assertExecuteUpdate[db -> MySQL -> Placeholder -> DROP ROLE role1](org.apache.shardingsphere.dbtest.engine.dcl.GeneralDCLIT)  Time elapsed: 3.505 s  <<< ERROR!
java.sql.SQLException: Operation DROP ROLE failed for 'role1'@'%'
	at org.apache.shardingsphere.dbtest.engine.dcl.GeneralDCLIT.assertExecuteUpdate(GeneralDCLIT.java:53)

[ERROR] assertExecute[db -> MySQL -> Placeholder -> DROP ROLE role1](org.apache.shardingsphere.dbtest.engine.dcl.GeneralDCLIT)  Time elapsed: 3.581 s  <<< ERROR!
java.sql.SQLException: Operation DROP ROLE failed for 'role1'@'%'
	at org.apache.shardingsphere.dbtest.engine.dcl.GeneralDCLIT.assertExecute(GeneralDCLIT.java:64)

[ERROR] assertExecuteUpdate[tbl -> MySQL -> Placeholder -> DROP ROLE role1](org.apache.shardingsphere.dbtest.engine.dcl.GeneralDCLIT)  Time elapsed: 1.721 s  <<< ERROR!
java.sql.SQLException: Operation DROP ROLE failed for 'role1'@'%'
	at org.apache.shardingsphere.dbtest.engine.dcl.GeneralDCLIT.assertExecuteUpdate(GeneralDCLIT.java:53)

[ERROR] assertExecute[tbl -> MySQL -> Placeholder -> DROP ROLE role1](org.apache.shardingsphere.dbtest.engine.dcl.GeneralDCLIT)  Time elapsed: 2.045 s  <<< ERROR!
java.sql.SQLException: Operation DROP ROLE failed for 'role1'@'%'
	at org.apache.shardingsphere.dbtest.engine.dcl.GeneralDCLIT.assertExecute(GeneralDCLIT.java:64)

[ERROR] assertExecuteUpdate[dbtbl_with_masterslave -> MySQL -> Placeholder -> DROP ROLE role1](org.apache.shardingsphere.dbtest.engine.dcl.GeneralDCLIT)  Time elapsed: 31.916 s  <<< ERROR!
java.sql.SQLException: Operation DROP ROLE failed for 'role1'@'%'
	at org.apache.shardingsphere.dbtest.engine.dcl.GeneralDCLIT.assertExecuteUpdate(GeneralDCLIT.java:53)

[ERROR] assertExecute[dbtbl_with_masterslave -> MySQL -> Placeholder -> DROP ROLE role1](org.apache.shardingsphere.dbtest.engine.dcl.GeneralDCLIT)  Time elapsed: 32.185 s  <<< ERROR!
java.sql.SQLException: Operation DROP ROLE failed for 'role1'@'%'
	at org.apache.shardingsphere.dbtest.engine.dcl.GeneralDCLIT.assertExecute(GeneralDCLIT.java:64)

[ERROR] assertExecuteUpdate[masterslave -> MySQL -> Placeholder -> DROP ROLE role1](org.apache.shardingsphere.dbtest.engine.dcl.GeneralDCLIT)  Time elapsed: 4.159 s  <<< ERROR!
java.sql.SQLException: Operation DROP ROLE failed for 'role1'@'%'
	at org.apache.shardingsphere.dbtest.engine.dcl.GeneralDCLIT.assertExecuteUpdate(GeneralDCLIT.java:53)

[ERROR] assertExecute[masterslave -> MySQL -> Placeholder -> DROP ROLE role1](org.apache.shardingsphere.dbtest.engine.dcl.GeneralDCLIT)  Time elapsed: 3.252 s  <<< ERROR!
java.sql.SQLException: Operation DROP ROLE failed for 'role1'@'%'
	at org.apache.shardingsphere.dbtest.engine.dcl.GeneralDCLIT.assertExecute(GeneralDCLIT.java:64)

[ERROR] assertExecuteUpdate[db -> MySQL -> Literal -> REVOKE SELECT ON  *.* FROM 'user1'@'localhost'](org.apache.shardingsphere.dbtest.engine.dcl.GeneralDCLIT)  Time elapsed: 3.668 s  <<< ERROR!
com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: There is no such grant defined for user 'user1' on host 'localhost'
	at org.apache.shardingsphere.dbtest.engine.dcl.GeneralDCLIT.assertExecuteUpdate(GeneralDCLIT.java:51)

[ERROR] assertExecute[db -> MySQL -> Literal -> REVOKE SELECT ON  *.* FROM 'user1'@'localhost'](org.apache.shardingsphere.dbtest.engine.dcl.GeneralDCLIT)  Time elapsed: 3.707 s  <<< ERROR!
com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: There is no such grant defined for user 'user1' on host 'localhost'
	at org.apache.shardingsphere.dbtest.engine.dcl.GeneralDCLIT.assertExecute(GeneralDCLIT.java:62)

[ERROR] assertExecuteUpdate[tbl -> MySQL -> Literal -> REVOKE SELECT ON  *.* FROM 'user1'@'localhost'](org.apache.shardingsphere.dbtest.engine.dcl.GeneralDCLIT)  Time elapsed: 1.666 s  <<< ERROR!
com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: There is no such grant defined for user 'user1' on host 'localhost'
	at org.apache.shardingsphere.dbtest.engine.dcl.GeneralDCLIT.assertExecuteUpdate(GeneralDCLIT.java:51)

[ERROR] assertExecute[tbl -> MySQL -> Literal -> REVOKE SELECT ON  *.* FROM 'user1'@'localhost'](org.apache.shardingsphere.dbtest.engine.dcl.GeneralDCLIT)  Time elapsed: 1.67 s  <<< ERROR!
com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: There is no such grant defined for user 'user1' on host 'localhost'
	at org.apache.shardingsphere.dbtest.engine.dcl.GeneralDCLIT.assertExecute(GeneralDCLIT.java:62)

[ERROR] assertExecuteUpdate[dbtbl_with_masterslave -> MySQL -> Literal -> REVOKE SELECT ON  *.* FROM 'user1'@'localhost'](org.apache.shardingsphere.dbtest.engine.dcl.GeneralDCLIT)  Time elapsed: 31.257 s  <<< ERROR!
com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: There is no such grant defined for user 'user1' on host 'localhost'
	at org.apache.shardingsphere.dbtest.engine.dcl.GeneralDCLIT.assertExecuteUpdate(GeneralDCLIT.java:51)

[ERROR] assertExecute[dbtbl_with_masterslave -> MySQL -> Literal -> REVOKE SELECT ON  *.* FROM 'user1'@'localhost'](org.apache.shardingsphere.dbtest.engine.dcl.GeneralDCLIT)  Time elapsed: 31.294 s  <<< ERROR!
com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: There is no such grant defined for user 'user1' on host 'localhost'
	at org.apache.shardingsphere.dbtest.engine.dcl.GeneralDCLIT.assertExecute(GeneralDCLIT.java:62)

[ERROR] assertExecuteUpdate[masterslave -> MySQL -> Literal -> REVOKE SELECT ON  *.* FROM 'user1'@'localhost'](org.apache.shardingsphere.dbtest.engine.dcl.GeneralDCLIT)  Time elapsed: 3.154 s  <<< ERROR!
com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: There is no such grant defined for user 'user1' on host 'localhost'
	at org.apache.shardingsphere.dbtest.engine.dcl.GeneralDCLIT.assertExecuteUpdate(GeneralDCLIT.java:51)

[ERROR] assertExecute[masterslave -> MySQL -> Literal -> REVOKE SELECT ON  *.* FROM 'user1'@'localhost'](org.apache.shardingsphere.dbtest.engine.dcl.GeneralDCLIT)  Time elapsed: 3.308 s  <<< ERROR!
com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: There is no such grant defined for user 'user1' on host 'localhost'
	at org.apache.shardingsphere.dbtest.engine.dcl.GeneralDCLIT.assertExecute(GeneralDCLIT.java:62)

[ERROR] assertExecuteUpdate[db -> MySQL -> Placeholder -> REVOKE SELECT ON  *.* FROM 'user1'@'localhost'](org.apache.shardingsphere.dbtest.engine.dcl.GeneralDCLIT)  Time elapsed: 3.599 s  <<< ERROR!
com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: There is no such grant defined for user 'user1' on host 'localhost'
	at org.apache.shardingsphere.dbtest.engine.dcl.GeneralDCLIT.assertExecuteUpdate(GeneralDCLIT.java:53)

[ERROR] assertExecute[db -> MySQL -> Placeholder -> REVOKE SELECT ON  *.* FROM 'user1'@'localhost'](org.apache.shardingsphere.dbtest.engine.dcl.GeneralDCLIT)  Time elapsed: 3.459 s  <<< ERROR!
com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: There is no such grant defined for user 'user1' on host 'localhost'
	at org.apache.shardingsphere.dbtest.engine.dcl.GeneralDCLIT.assertExecute(GeneralDCLIT.java:64)

[ERROR] assertExecuteUpdate[tbl -> MySQL -> Placeholder -> REVOKE SELECT ON  *.* FROM 'user1'@'localhost'](org.apache.shardingsphere.dbtest.engine.dcl.GeneralDCLIT)  Time elapsed: 1.665 s  <<< ERROR!
com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: There is no such grant defined for user 'user1' on host 'localhost'
	at org.apache.shardingsphere.dbtest.engine.dcl.GeneralDCLIT.assertExecuteUpdate(GeneralDCLIT.java:53)

[ERROR] assertExecute[tbl -> MySQL -> Placeholder -> REVOKE SELECT ON  *.* FROM 'user1'@'localhost'](org.apache.shardingsphere.dbtest.engine.dcl.GeneralDCLIT)  Time elapsed: 1.902 s  <<< ERROR!
com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: There is no such grant defined for user 'user1' on host 'localhost'
	at org.apache.shardingsphere.dbtest.engine.dcl.GeneralDCLIT.assertExecute(GeneralDCLIT.java:64)

[ERROR] assertExecuteUpdate[dbtbl_with_masterslave -> MySQL -> Placeholder -> REVOKE SELECT ON  *.* FROM 'user1'@'localhost'](org.apache.shardingsphere.dbtest.engine.dcl.GeneralDCLIT)  Time elapsed: 30.663 s  <<< ERROR!
com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: There is no such grant defined for user 'user1' on host 'localhost'
	at org.apache.shardingsphere.dbtest.engine.dcl.GeneralDCLIT.assertExecuteUpdate(GeneralDCLIT.java:53)

[ERROR] assertExecute[dbtbl_with_masterslave -> MySQL -> Placeholder -> REVOKE SELECT ON  *.* FROM 'user1'@'localhost'](org.apache.shardingsphere.dbtest.engine.dcl.GeneralDCLIT)  Time elapsed: 32.218 s  <<< ERROR!
com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: There is no such grant defined for user 'user1' on host 'localhost'
	at org.apache.shardingsphere.dbtest.engine.dcl.GeneralDCLIT.assertExecute(GeneralDCLIT.java:64)

[ERROR] assertExecuteUpdate[masterslave -> MySQL -> Placeholder -> REVOKE SELECT ON  *.* FROM 'user1'@'localhost'](org.apache.shardingsphere.dbtest.engine.dcl.GeneralDCLIT)  Time elapsed: 3.107 s  <<< ERROR!
com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: There is no such grant defined for user 'user1' on host 'localhost'
	at org.apache.shardingsphere.dbtest.engine.dcl.GeneralDCLIT.assertExecuteUpdate(GeneralDCLIT.java:53)

[ERROR] assertExecute[masterslave -> MySQL -> Placeholder -> REVOKE SELECT ON  *.* FROM 'user1'@'localhost'](org.apache.shardingsphere.dbtest.engine.dcl.GeneralDCLIT)  Time elapsed: 3.585 s  <<< ERROR!
com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: There is no such grant defined for user 'user1' on host 'localhost'
	at org.apache.shardingsphere.dbtest.engine.dcl.GeneralDCLIT.assertExecute(GeneralDCLIT.java:64)

[INFO] Running org.apache.shardingsphere.dbtest.engine.ddl.GeneralDDLIT
[INFO] ======= Test Plan =======
[INFO] Total: 0, Active: 0, Disabled: 0 

[INFO] Tests run: 0, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0 s - in org.apache.shardingsphere.dbtest.engine.ddl.GeneralDDLIT
[INFO] Running org.apache.shardingsphere.dbtest.engine.dml.AdditionalDMLIT
[INFO] Tests run: 0, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0 s - in org.apache.shardingsphere.dbtest.engine.dml.AdditionalDMLIT
[INFO] Running org.apache.shardingsphere.dbtest.engine.dml.BatchDMLIT
[INFO] ======= Test Plan =======
[WARN] Total: 832, Active: MySQL(240), Disabled: H2(152), Oracle(160), SQLServer(136), PostgreSQL(144) 
```

--- a/shardingsphere-integration-test/shardingsphere-test-suite/src/test/resources/integrate/cases/dcl/dcl-integrate-test-cases.xml
+++ b/shardingsphere-integration-test/shardingsphere-test-suite/src/test/resources/integrate/cases/dcl/dcl-integrate-test-cases.xml
@@ -16,7 +16,7 @@
   -->
 <integrate-test-cases>
     <dcl-test-case sql="CREATE USER 'user_dev_new'@'localhost'" db-types="MySQL" />
-    <dcl-test-case value="CREATE USER user_dev_new" db-types="PostgreSQL" />
+    <dcl-test-case sql="CREATE USER user_dev_new" db-types="PostgreSQL" />
     <dcl-test-case sql="CREATE USER 'user_dev_new'@'localhost' identified by 'passwd_dev'" db-types="MySQL"/>
     <dcl-test-case sql="CREATE USER user_dev_new identified by passwd_dev" db-types="Oracle" />
     <dcl-test-case sql="CREATE USER user_dev_new FOR LOGIN login_dev" db-types="SQLServer"/>
@@ -38,7 +38,7 @@
     <dcl-test-case sql="ALTER USER user_dev WITH NAME = user_dev_new" db-types="SQLServer"/>
     <dcl-test-case sql="ALTER USER user_dev RENAME TO user_dev_new" db-types="PostgreSQL"/>
     <dcl-test-case sql="ALTER USER user_dev WITH PASSWORD = 'passwd_dev'" db-types="SQLServer"/>
-    <dcl-test-case value="ALTER USER user_dev WITH PASSWORD 'passwd_dev'" db-types="PostgreSQL"/>
+    <dcl-test-case sql="ALTER USER user_dev WITH PASSWORD 'passwd_dev'" db-types="PostgreSQL"/>
     <dcl-test-case sql="ALTER USER user_dev WITH PASSWORD 'passwd_dev'" db-types="PostgreSQL"/>
     <dcl-test-case sql="ALTER ROLE role1_bak DROP MEMBER member1" db-types="SQLServer"/>
     <dcl-test-case sql="ALTER USER CURRENT_USER WITH ENCRYPTED PASSWORD 'password'" db-types="PostgreSQL"/>

